reference/channels.v3.yml

```diff
@@ -502,11 +502,118 @@ paths:
           $ref: '#/components/responses/single_listing_resp'
         '404':
           $ref: '#/components/responses/listing_not_found_resp'
+  '/stores/{store_hash}/v3/channels/{channel_id}/site/checkout-url':
+    parameters:
+      - name: channel_id
+        description: The ID of a channel
+        in: path
+        required: true
+        schema:
+          type: integer
+          format: int64
+      - schema:
+          type: string
+        name: store_hash
+        in: path
+        required: true
+    put:
+      summary: Upsert a Site's Checkout URL
+      tags:
+        - Channel Site Checkout Url
+      description: Creates or updates (upserts) a site's checkout URL
+      operationId: put-checkout-url
```

Comment:
I think I've seen operationIds be CamelCased elsewhere... is this kebab thing what we want?